dbt_project.yml

```diff
@@ -21,9 +21,17 @@ vars:
     field: "{{ source('jira', 'field') }}"
     field_option: "{{ source('jira', 'field_option') }}"
     sprint: "{{ source('jira', 'sprint') }}"
+    team: "{{ source('jira', 'team') }}"
     version: "{{ source('jira', 'version') }}"
     jira_issue_history_buffer: 1
 
+analysis-paths: ["analysis"]
+clean-targets:
+  - target
+  - dbt_modules
+  - dbt_packages
+
```

Comment:
Let's not add the clean-targets as this is unnecessary. We should keep the analysis path though.
```suggestion
```

Reply:
removed